vertx-mongo-client/src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java

```diff
@@ -100,4 +92,8 @@ public MongoClientSettings settings() {
   public String database() {
     return database;
   }
+
+  private Optional<ReadConcern> maybeReadConcern(ConnectionString connectionString, JsonObject config) {
```

Comment:
Maybe rename this to `setReadConcernIfExists` to be clearer.  

Also see Brian Goetz's guidance on using Optional in place of null checks https://stackoverflow.com/questions/26327957/should-java-8-getters-return-optional-type/26328555

Reply:
But it doesn't set anything, it just returns something that could be absent. What's your suggestion with that stackoverflow question exactly? It's not just a dummy getter.